FocusOn/Services/DataServiceProtocol.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import Foundation
+import SwiftUICharts
```

Comment:
Theoretically, your data services should not depend of a UI libraries such as `SwiftUICharts`.
This is thanks to the parsing done by the view models, it prevents the UI to know about `CoreData`, `API` or anything too technical, and it prevents the services to know about `SwiftUI`, `UIKit` or any anything too technical.